src/main/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExample.java

```diff
@@ -323,34 +332,35 @@ static KafkaStreams createChartsStreams(final String bootstrapServers,
     // store "top-five-songs-by-genre", and this state store can then be queried interactively via a REST API (cf.
     // MusicPlaysRestService) for the latest charts per genre.
     songPlayCounts.groupBy((song, plays) ->
-            KeyValue.pair(song.getGenre().toLowerCase(),
-                new SongPlayCount(song.getId(), plays)),
-        Serdes.String(),
-        songPlayCountSerde)
-        // aggregate into a TopFiveSongs instance that will keep track
-        // of the current top five for each genre. The data will be available in the
-        // top-five-songs-genre store
-        .aggregate(TopFiveSongs::new,
-            (aggKey, value, aggregate) -> {
-              aggregate.add(value);
-              return aggregate;
-            },
-            (aggKey, value, aggregate) -> {
-              aggregate.remove(value);
-              return aggregate;
-            },
-            topFiveSerde,
-            TOP_FIVE_SONGS_BY_GENRE_STORE
-        );
+                                   KeyValue.pair(song.getGenre().toLowerCase(),
```

Comment:
The indentations here (and elsewhere in the PR) look pretty odd.  Is this really auto-formatted?

e.g. sometimes we indent with 2 spaces, sometimes 4 or even 8?

Reply:
Yeah, i have no idea what is going on with the auto-formatting. I think it is likely because we use different styles for AK and confluent. Which is pretty frustrating

Reply:
We do have -- but you can configure your IDE with different setting for different project.